examples/protocol_specification_ex/tac.yaml

```diff
@@ -0,0 +1,45 @@
+name: tac
+author: fetchai
+version: 0.1.0
+license: Apache 2.0
+description: 'A protocol for participating in a TAC.'
+speech_acts:
+  register:
+    agent_name: str
+  unregister: {}
+  transaction:
+    tx_id: str
+    tx_sender_addr: str
+    tx_counterparty_addr: str
+    amount_by_currency_id: dict
```

Comment:
`Dict[str, int]` etc.. for all needs to be specified

Reply:
Done